barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover_ultra.cpp

```diff
@@ -0,0 +1,8 @@
+// Note: this is split up from protogalaxy_prover_impl.hpp for compile performance reasons
+#include "protogalaxy_prover_impl.hpp"
+
+namespace bb {
+template class ProtoGalaxyProver_<ProverInstances_<UltraFlavor, 2>>;
+template class ProtoGalaxyProver_<ProverInstances_<UltraFlavor, 3>>;
+template class ProtoGalaxyProver_<ProverInstances_<UltraFlavor, 4>>;
```

Comment:
we could probably disable the 2 and 3 ones for now if its impacting compilation time